modules/osdk-init-resource.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-init-resource_{context}"]
+= Providing templates for required custom resources
+
+An Operator might require the user to instantiate a custom resource before the
+Operator can be fully functional. However, it can be challenging for a user to
+determine what is required or how the resource should be defined.
+
+Using the `operators.operatorframework.io/internal-objects` annotation in the
+ClusterServiceVersion (CSV), an Operator developer can specify a required custom
+resource that should be created at the time that the Operator is installed. A
+template can also be provided in the CSV for the resource:
```

Comment:
"A template _can also_ be provided ..." worries me a bit if this makes it sound like an option but it's in fact a must-have. Is it possible to tweak this a bit? thanks!